packages/data-wizard/package.json

```diff
@@ -50,16 +50,19 @@
   "limit-size": [
     {
       "path": "dist/index.min.js",
-      "limit": "10 Kb",
+      "limit": "70 Kb",
       "gzip": true
     },
     {
       "path": "dist/index.min.js",
-      "limit": "30 Kb"
+      "limit": "200 Kb"
     }
   ],
   "dependencies": {
-    "@antv/ckb": "^2.0.0-alpha.1"
+    "@antv/ckb": "^2.0.0-alpha.1",
```

Comment:
wait, why ckb...

Reply:
We `import type { LevelOfMeasurement } from '@antv/ckb'` at analyzer. 🤔 but, why it's deleted at diff and added again.

Reply:
，